client/blocks/reader-full-post/index.jsx

```diff
@@ -243,6 +238,22 @@ export class FullPostView extends React.Component {
 		}
 	}
 
+	goToNextPost = () => {
+		if ( getLastStore() ) {
```

Comment:
I thought about this, and even had it that way initially.
1. It doesn't belong in state (shouldnt trigger render, state should be minimized)
2. we could attach it to something like `this.store = ...` but I don't think thats actually better.  Its not a perf boon since this is a basic getter

That being said, I don't have as fresh eyes on this. Do you think it would be easier to understand/follow if I set `this.store = getLastStore()` within `componentDidMount`?  Then `goToNextPost` would have: 
`if ( this.store ) { ... }`